clarify use of bold text with UI elements (#14992)
cockroachdb/docs: StyleGuide.md

#### Bold

Use bold text to emphasize an important word or phrase, or to create visual separation and callouts (e.g., **Example:**). Do not combine bold with italic.

> Bold text should be used whenever referring to the name of a UI section or field. The name should only be in bold if it appears verbatim in the UI. Some UI sections, such as tables, are not labeled in the UI and should not be bolded when named in the documentation.

Reviewer: ```suggestion
Use bold text you refer to the name of a UI section or field. The name should be in bold only if it appears verbatim in the UI. If a UI element, such as a table, is not labeled in the UI, do not bold when you reference the element.
```

Author: I'd like to further edit and use your suggestion as follows. Look ok?

```suggestion
Use bold text when you refer to the name of a UI section or field. The name should be in bold only if it appears verbatim in the UI. If a UI element, such as a table, is not labeled in the UI, do not bold when you reference the element in the documentation.
```

Reviewer: LGTM
